balloc/balloc.c

```diff
@@ -957,12 +957,13 @@ static int balloc_init_internal(struct m0_balloc *bal,
 				m0_bcount_t spare_blocks_per_group)
 {
 	int rc;
-
 	M0_ENTRY();
 
 	bal->cb_be_seg = seg;
 	bal->cb_group_info = NULL;
 	m0_mutex_init(&bal->cb_sb_mutex.bm_u.mutex);
+	M0_LOG(M0_ERROR,"YB:Balloc key size 1:%"PRIu64"",gd_tree_key_size(NULL));
```

Comment:
Is it for debug purpose and will be removed in final commit ? If not please remove `YB:`

Reply:
Yes. It is for debug purpose and it will be remove in final commit.